chore: Add docs for capabilities value length limitation (#2336)
saucelabs/sauce-docs: docs/dev/test-configuration-options.md

- `removeApp`: Managed by RDC differently, but cannot be used inside an Appium test as part of the routine.
- `Edit Timezone`: Appium does not provide a capability to edit the timezone of a device in automated testing on real devices.
- See [Virtual Device Capabilities](#virtual-device-capabilities-sauce-specific--optional) for information about timezone capabilities in a virtual device testing.

> ## Limitations

Reviewer: ```suggestion
```

> :::warning

Reviewer: ```suggestion
:::caution Limitations
```

> When running a test on a Virtual Device, please be aware that there is a 100 characters limitation for each capability value. If the value exceeds this limit, it will be truncated, which can lead to further side effects or prevent a job from starting.

Reviewer: ```suggestion
When running a test on a Virtual Device, be aware that each capability value has a 100 characters limitation. If the value exceeds this limit, it will be truncated, which can lead to further side effects or prevent a job from starting.
```

> When running a test on a Virtual Device, please be aware that there is a 100 characters limitation for capabilities value. If the value exceeds this limit, it will be truncated and leading to issues. Make sure the value does not exceed the limitation to avoid any issues.

Reviewer 2: ```suggestion
When running a test on a Virtual Device, please be aware that there is a 100 characters limitation for each capability value. If the value exceeds this limit, it will be truncated, which can lead to further side effects or prevent a job from starting.
```
